data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -21,24 +23,49 @@ t_distinct_hypothesis_uri_doi_version_with_elife_doi_version AS (
     *,
     ROW_NUMBER() OVER(
       PARTITION BY source_doi
-      ORDER BY source_version
+      ORDER BY source_doi_version
     ) AS elife_doi_version
   FROM t_distinct_hypothesis_uri_doi_version
 ),
 
+t_distinct_hypothesis_uri_id_and_timestamp AS(
+  SELECT 
+    DISTINCT
+    uri,
+    source_doi,
+    source_doi_version,
+    annotation_created_timestamp,
+    hypothesis_id
+  FROM t_hypothesis_annotation_with_doi
+),
+
+t_distinct_hypothesis_with_peer_review_suffix_number AS (
+  SELECT
+    *,
+    ROW_NUMBER() OVER(
+      PARTITION BY source_doi, source_doi_version
+      ORDER BY annotation_created_timestamp, hypothesis_id
+    ) AS peer_review_suffix_number
```

Comment:
Just wondering whether the name `peer_review` could be misleading as it also contains evaluation summary and author responses. The term used before was `evaluation` (although granted that could be misleading too but is what was also used within Sciety).